src/main/java/com/corndel/pixmate/drawings/TriangleSandwich.java

```diff
@@ -0,0 +1,27 @@
+package com.corndel.pixmate.drawings;
+
+public class TriangleSandwich {
+    public static void main(String[] args) {
+
+        int height = 5;
+
+        for (int i = 0; i <= height; i++) {
+            StringBuilder line = new StringBuilder();
+            for (int j = 0; j < i; j++) {
+                line.append("*");
+
```

Comment:
With this you would probably only want to use one StringBuilder, if you move it outside of the initial for loop scope both for loops will be able to get access to it to append the *'s.

```java
package com.corndel.pixmate.drawings;

public class TriangleSandwich {
    public static void main(String[] args) {

        int height = 5;
        StringBuilder line = new StringBuilder();

        for (int i = 0; i <= height; i++) {
            for (int j = 0; j < i; j++) {
                line.append("*");
            }
        }

        for (int i = height - 1; i > 0; i--) {
            for (int j = 0; j < i; j++) {
                line.append("*");
            }
        }
        System.out.println(line);
    }
}

```